tools/loop-mq/_mixin.scss

```diff
@@ -13,6 +14,37 @@ Auto-generates media queries for looping content
 }
 ```
 
+**Use with a prop**
+```scss
+.my-class {
+  @include loop-mq {
+    margin: 1rem;
+  }
+}
+```
+
+**CSS Output**
+```css
```

Comment:
This doesn't feel like a great example of this. Suggestions?

Reply:
I think this good to show what the class looks like in the CSS, but I think what would help is an explanation that in the HTML the `@` does not need to be escaped as it is in the CSS.